docs/source/features/parallelisms.rst

```diff
@@ -80,10 +72,36 @@ The distributed optimizer in NeMo is built on top of
 `DistributedFusedAdam <https://github.com/NVIDIA/apex/blob/master/apex/contrib/optimizers/distributed_fused_adam.py>`_
 from Apex.
 
+Fully-Shared Data Parallelism (FSDP)
```

Comment:
Edits to lines 66 - 69

In NeMo, Distributed Data Parallelism (DDP) can use either the default PyTorch DistributedDataParallel or a custom implementation (when custom multi-precision training is enabled with megatron_amp_O2).